examples/dht_sensor/Makefile

```diff
@@ -0,0 +1,4 @@
+
+PROGRAM=dht_sensor
+COMPONENTS = FreeRTOS lwip core extras/dht
```

Comment:
I've been meaning to go through and fix this in our existing examples (so it's completely understandable that you didn't know this) but instead of overriding `COMPONENTS` here, you should instead just set `EXTRA_COMPONENTS = extras/dht` (this will make it work properly even if we change the list of default core components later)